README.md

```diff
@@ -102,6 +122,7 @@ adapter patterns in use today.
     BindExpression[Yield] :
         LeftHandSideExpression[?Yield] :: [lookahead ≠ new] MemberExpression[?Yield]
         :: MemberExpression[?Yield]
+        LeftHandSideExpression[?Yield] :: new
```

Comment:
I think so, because otherwise it will create some ASI hazards:

```js
C::new
foo()
```

Without the restriction, this would parse as `C::new foo()`, which is not what is intended.